freezeyt/cli.py

```diff
@@ -1,16 +1,20 @@
-import importlib
-
 import click
+import importlib
+import yaml
 
 from freezeyt.freezing import freeze
```

Comment:
1. Standardní knihovna
2. 3rd party
3. Knihovny z tohoto projektu

```suggestion
import importlib

import click
import yaml

from freezeyt.freezing import freeze
```